packaging/debian/syslog-ng-mod-kafka.install

```diff
@@ -1,3 +1,2 @@
 usr/lib/syslog-ng/*/java-modules/kafka.jar
 usr/share/syslog-ng/include/scl/kafka/*
```

Comment:
the kafka scl needs to go to the core package in this case, as one would need to install both rdkafka and the java based kafka implementation to get the switching between the two implementations.

Reply:
I don't fully understand the switching part (I've seen the `kafka-implementation` define and subst), but if `scl/kafka/plugin.conf` will be moved to the core package and the default implementation is the Java version, then a `@requires` pragma should be added for Java.

Reply:
To be able to have the switching between the implementations as @bazsi said, I guess the kafka SCL has to be split then, right?
We cannot put the `kafka-java` part into the core, as then the core will depend on java also.
What about putting it into `scl/scl.conf` file? It already contains `java-module-dir` definition.